### DISCUSSION TITLE
请问在WTF Academy里学完了一章并完成测试后为什么成绩还是显示0%呢？

### DISCUSSION THREAD

**Discussion by SKYeve** [AmazingAng/WTF-Solidity#233] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/55940004/194112051-2f131fbb-6194-49d4-b97e-0a8b87860a8c.png)

**Comment by AmazingAng** [comment, marked as answer]
看你右上角是登陆了的，那应该就是答题时没有填对github主邮箱。
可以在github用户页面查询自己的邮箱，填写好后就会显示了。

**Comment by SKYeve** [comment]
明白了，我以为是随便填一个邮箱呢，谢谢！

**Comment by Dani9541L** [comment]
![Screenshot (464)](https://user-images.githubusercontent.com/37637923/194695766-ea467077-3f91-4f9f-aa72-7656142ccc25.png)
我也遇到了同样的问题。我发觉我注册的时候，是影藏了我的邮箱账号。今天我把我的github账号改成可见的，public。然后填作业表的第一行也是填的github账号所示的同一邮箱，但是看版面，好像还是没有半点反应？

**Reply by Dani9541L** [reply]
已经解决了这个问题了。现在在填作业表的邮箱时，必须填注册WTF solidity时用户的github账号的primary 主邮箱账号，才能看到结果。
